src/Components/Components/src/CascadingValue.cs

```diff
@@ -110,7 +110,7 @@ public Task SetParametersAsync(ParameterCollection parameters)
             // because it serves no useful purpose to have a <CascadingValue> otherwise.
             if (!hasSuppliedValue)
             {
-                throw new ArgumentException($"Missing required parameter '{nameof(Value)}' for component '{nameof(Parameter)}'.");
+                throw new ArgumentException($"Missing required parameter '{nameof(Value)}' for component '{nameof(ParameterValue)}'.");
```

Comment:
I think this is a pre-existing bug. Should be:

```suggestion
                throw new ArgumentException($"Missing required parameter '{nameof(Value)}' for component '{nameof(CascadingValue)}'.");
```